pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchClient.java

```diff
@@ -96,6 +110,12 @@ public void afterBulk(long executionId, List<BulkProcessor.BulkOperationRequest>
         this.client = retry(() -> RestClientFactory.createClient(config, bulkListener), -1, "client creation");
     }
 
+    public void incrementCounter(String counter, double value) {
+        if (sinkContext != null && counter != null) {
```

Comment:
What's with the if? Why would both ever be null?

Reply:
Again, to evaluate sinkContext against null exists in KinesisSink.java. I will remove counter evaluation part.

Reply:
Can you please explain what is the relation of `sinkContext != null` to `KinesisSink` in this method/line?

Reply:
I removed the if evaluation.